testplan/runnable/interactive/http.py

```diff
@@ -906,3 +915,11 @@ def run(self):
             self._server.serve()
         finally:
             self._server = None
+
+    def stop(self):
```

Comment:
Shall we call it teardown instead?

Reply:
Actually the http handler is created by interactive handler and set it as parent, so I can change it to `aborting`, in `abort_dependencies` of interactive handler, yield that http handler.